### PR TITLE
Landryb/geos 7457

### PR DESCRIPTION
**Pull request by landryb**

### CODE REVIEW

**Comment by aaime** [comment]
@jodygarnett the dependency resolution keeps on making Travis builds fail:

````
[ERROR] Failed to execute goal on project gs-libjpeg-turbo: Could not resolve dependencies for project org.geoserver.extension:gs-libjpeg-turbo:jar:2.9-SNAPSHOT: Failed to collect dependencies at it.geosolutions.imageio-ext:imageio-ext-turbojpeg:jar:1.1.13 -> org.libjpegturbo:turbojpeg-wrapper:jar:1.2.1.1: Failed to read artifact descriptor for org.libjpegturbo:turbojpeg-wrapper:jar:1.2.1.1: Could not find artifact it.geosolutions.imageio-ext:imageio-ext-library:pom:1.2-SNAPSHOT in boundless (https://repo.boundlessgeo.com/main/) -> [Help 1]
[ERROR] Failed to execute goal on project gs-monitor-core: Could not resolve dependencies for project org.geoserver.extension:gs-monitor-core:jar:2.9-SNAPSHOT: Could not find artifact com.maxmind:geoip:jar:1.2.3 in boundless (https://repo.boundlessgeo.com/main/) -> [Help 1]
[ERROR] Failed to execute goal org.apache.maven.plug
````

I'll have a look at the libjpegturbo thing, it's weird that a SNAPSHOT is even taken into consideration, but the geoip lookup failure appears to be a legit problem about the boundless repo. 
Looks like it's an old jar that is not in central anymore, but used to be in the old repo.opengeo.org

**Comment by landryb** [comment]
Iirc we've had issues with boundless maven repos within georchestra, @pmauduit might remember more.

**Comment by jodygarnett** [comment]
I think repo issues are now resolved, @landryb can you update the headers on the files you changed? It would allow me to merge this in :)

**Comment by landryb** [comment]
@jodygarnett  Update the headers ? What do you mean ? merge master in the PR ?

**Comment by aaime** [comment]
Hi @landryb, when you change a file you have to update its copyright header. The contribution process, including copyright updates ,is available here: https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md
Github links to that document whenever one makes a pull request, but while the link is on the top of the page, it's also easy not to notice.

File: src/extension/importer/core/src/main/java/org/geoserver/importer/format/GeoJSONFormat.java [hunk 1]
```diff
@@ -159,7 +159,7 @@ ImportTask task(ImportData data, Catalog catalog) throws IOException {
 
         // get the composite feature type
         SimpleFeatureType featureType = new FeatureJSON().readFeatureCollectionSchema(file.file(), false);
-        System.out.println(featureType);
+        LOG.log(Level.DEBUG, featureType);
```
Comment (aaime) [inline src/extension/importer/core/src/main/java/org/geoserver/importer/format/GeoJSONFormat.java]: Travis reports a compile error here:

````
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/travis/build/geoserver/geoserver/src/extension/importer/core/src/main/java/org/geoserver/importer/format/GeoJSONFormat.java:[162,21] error: cannot find symbol
[INFO] 1 error
````

Hum... I'd say the problem is that DEBUG is a Log4J level, but the API we are using is Java logging (even if it then delegates to Log4J under covers)

**Comment by jodygarnett** [comment]
@landryb can you rebase against master (to update this branch and resolve conflicts) please?

**Comment by landryb** [comment]
Rebased on top of master, travis failed but the log is just a huge vomit

**Comment by landryb** [comment]
Ah:
```
[ERROR] /home/travis/build/geoserver/geoserver/src/extension/importer/core/src/main/java/org/geoserver/importer/format/GeoJSONFormat.java:[161,11] error: no suitable method found for log(Level,SimpleFeatureType)
```

Guess i'll fix that one..

**Comment by landryb** [comment]
So a SimpleFeatureType can be serialized/stringified by System.out.println but not by LOG.log ? What should be used then ?

**Comment by mprins** [comment]
@landryb you could explicitly call `.toString()` on the SimpleFeatureType or coerce to String using `""+ SimpleFeatureType`

**Comment by landryb** [comment]
Thanks @mprins , that's what i realized after finally finding out the SimpleFeatureTypeImpl doc on http://docs.geotools.org/stable/javadocs/org/geotools/feature/simple/SimpleFeatureTypeImpl.html

**Comment by landryb** [comment]
Finally, after several tries, travis is happy with the last commits, so should be good to merge.

**Comment by aaime** [comment]
Squashed the commits back into one and merged on master and 2.9.x 
Now checking a build for a backport to 2.8.x as well.

**Comment by aaime** [comment]
Err no, fails to compile on 2.8.x... oh well, gotta run, kids are waking up

**Pull request closed without merging**